examples/with-npm/.npmrc

```diff
@@ -0,0 +1 @@
+auto-install-peers = true
```

Comment:
is this new? is this based on npm version that it was generated with? is this a next.js addition?

Reply:
Think it was there for historical ESLint-resolving reasons but shouldn't need it anymore.